### DISCUSSION TITLE
NSPANEL flash Tasmota

### DISCUSSION THREAD

**Discussion by Ilmessaggeromascherato** [arendst/Tasmota#22641] [Support request]
Hello 
I would like to install in my home NSPanel to manage temperature. 
I am able to flash it to TASMOTA, after that I see Tasmota wifi and I able to configure Wi-FI Connection and connect to web environment. 
In this moment NSPanle display don't show w nothing. So I followed instruction where I need to up load spaniel.be after that I lost connection whit NSPANEL and the only thing that I possible is refresh again. Could you help me to complete flash Tasmota in NSpanel ? Is there e manual, documentation to follow to be sure to complete this action ? I have a lot of Sonoff device all with Tasmota, I have this problem with nspanel. Some one can help me ? 
Thank you 
Stefano

**Comment by sfromis** [comment]
https://templates.blakadder.com/sonoff_NSPanel.html is where the instructions for flashing is. That does not mention any `spaniel.be` file, but does suggest `nspanel.be`.

Without details around "lost connection", hard to make any suggestion there, but do check if it is broadcasting an AP for you to connect to. You can also use the same serial connection used for flashing to connect a serial terminal program to see what Tasmota is doing.

Do note that many USB/serial adapters cannot supply the power for connecting to Wifi after Tasmota has been flashed, with you instead needing mains power, and disconnecting the adapter.
